tests/utils/test_process_util.py

```diff
@@ -122,21 +163,43 @@ def test_process_stdout_stderr(self):
         actual = capture_from_process(process, cmd)
         self.assertEqual(expected, actual)
 
-    def test_process_timeout_non_forked(self):
+    def test_it_should_throw_if_process_is_not_leader_and_timeout_is_given(self):
```

Comment:
this is not a new test... it's an existing test (was test_process_bad_pgid) that I moved here so that it is adjacent to the related 'test_it_should_capture_the_output_if_process_is_not_leader_and_timeout_is_not_given'